games/o.yaml

```
@@ -17,6 +17,31 @@
   type: remake
   updated: 2025-05-19
 
+- name: Oblast
+  originals:
+  - Blasto
+  type: clone
+  repo: 'https://github.com/classilla/oblast'
+  url: >-
+    https://oldvcr.blogspot.com/2025/12/oblast-better-blasto-game-for-commodore.html
+  feed: 'https://github.com/classilla/oblast/releases.atom'
+  development: very active
+  status: playable
+  multiplayer:
+  - Local
+  content: open
+  langs:
+  - Perl
+  - QuickBASIC
```

```suggestion
  - Assembly
  - BASIC
```
this is a C64 native game so this is completely wrong